rcongui/src/components/LogsView/logs.js

```diff
@@ -132,6 +180,11 @@ class Logs extends React.Component {
     localStorage.setItem("logs_limit", limit);
   }
 
+  sethighlightLogs(highlightLogs) {
```

Comment:
shouldn't this be camel case: setHighlightLogs

Reply:
Yes, I missed this. The coding standard at work uses snake_case, making me somewhat allergic to camelCase.

Reply:
Fixed.